packages/metals-languageclient/src/getJavaConfig.ts

```diff
@@ -40,7 +40,6 @@ export function getJavaConfig({
   const extraEnv = {
     ...coursierRepositories,
     ...coursierMirrors,
-    JAVA_HOME: javaHome,
```

Comment:
Maybe we should still set it if not available?

Reply:
It shouldn't be needed, since we always fallback to checking system properties java home.

Reply:
We can change that if it turns out to be a problem sure